### ISSUE TITLE
ui error page doesn't honor the slack settings in config

### ISSUE THREAD

**Issue by sahilvv** [temporalio/ui#609]
**Describe the bug**
Temporal UI [error page](https://github.com/temporalio/ui/blob/278c683c091418f284296bfb6dedcb666f11db7e/src/lib/components/error.svelte#L47) doesn't honor the slack settings in the config. 

**To Reproduce**

1. break the ES integration with frontend pod by feeding wrong credentials or not creating indexes
2. load UI page and you will see 500 error page and in that there will be a link to slack

**Expected behavior**
href should use the slack url present in the config.

**Screenshots**

**Desktop (please complete the following information):**

- OS: [macOS]
- Browser chrome
- Version [2.0.1]

**Additional context**

**Comment by sahilvv** [comment]
@feedmeapples sorry should have been a bit more explicit.

https://github.com/temporalio/ui/pull/519/files introduced FeedbackURL in the settings. I meant can we use this on the errorPage as well.

**Comment by feedmeapples** [comment]
right, @canvanooo should we show the `Feedback` button on the error page as well? 
![image](https://user-images.githubusercontent.com/11838981/174872542-c5517925-ed26-40c0-ab68-b80e3b38cc27.png)
